README.md

```diff
@@ -163,6 +123,64 @@ You can also call `sbot.metafeeds.ensureLoaded(feedId, cb)` on an individual
 basis to make sure that `findByIdSync` will operate at the correct time when the
 `feedId`'s metadata has been processed in the local database.
 
+### 🌷 `sbot.metafeeds.branchStream(opts)`
+
+Returns a **[pull-stream] source** of all "branches" in the meta feed trees.
+
+A "branch" is an array where the first item is the root meta feed and the
+subsequent items are the children and grandchildren of the root. A branch looks
+like this:
+
+```js
+[
+  [rootMetafeedId, null],
+  [childMetafeedId, childDetails],
+  [grandchildMetafeedId, grandchildDetails],
+  [grandgrandchildSubfeedId, grandgrandChildDetails],
+]
+```
+
+Or in general, an `Array<[FeedId, Details | null]>`. **details** if the object
```

Comment:
Something wrong with this sentence. **details** if? Maybe something like: **details** if defined will contain ... like ...

Reply:
Ah, `if` => `is`.